clients/scanoss/src/main/kotlin/model/Model.kt

```diff
@@ -67,6 +67,21 @@ data class License(
     val osadlUpdated: String? = null
 )
 
+/**
+ * The "copyrights" section of the raw report.
+ */
+@Serializable
+data class Copyright(
+    /** Copyright found. */
```

Comment:
Just "Copyright found" sounds as if this was a boolean (that indicated whether a copyright was found). Better say "*The* copyright found".